doc/api/util.md

```diff
@@ -326,9 +327,12 @@ changes:
 * `constructor` {Function}
 * `superConstructor` {Function}
 
+> Stability: 0 - Deprecated: Use `class` with [`extends`][] instead.
+
 Usage of `util.inherits()` is discouraged. Please use the ES6 `class` and
-`extends` keywords to get language level inheritance support. Also note
-that the two styles are [semantically incompatible][].
+[`extends`][] keywords to get language level inheritance support or use
+[`Object.setPrototypeOf()`]. Also note that the two styles are [semantically
+incompatible][].
```

Comment:
I think we should explain what this means here.

Reply:
I updated the documentation to state the meaning much clearer. PTAL.